django_project/ta_system/forms.py

```diff
@@ -23,6 +26,44 @@ class SemesterForm(forms.Form):
 
 
 class ApplicationForm(forms.Form):
+
+    def get_grad_years():
+        year = date.today().year
+        grad_years = [(year, str(year))]
+        for i in range(4):
+            year += 1
+            grad_years.append((year, str(year)))
+        return grad_years
+
+    course_choices = [('', 'No preference')]
+    sem = get_year_and_semester_code(get_current_semester())
+    current_semester = Semester.objects.filter(year=sem[0], semester_code=sem[1])[0]
```

Comment:
there is a function in utils called get_current_semester(), how is this one different than that one